src/tests/ftest/nvme/enospace.py

```diff
@@ -104,7 +105,7 @@ def ior_bg_thread(self, results):
 
         # Define the job manager for the IOR command
         self.job_manager = Mpirun(ior_bg_cmd, mpitype="mpich")
-        self.create_cont()
+        self.create_cont(dfs_oclass)
```

Comment:
(pylint-too-many-function-args) Too many positional arguments for method call
(pylint-undefined-variable) Undefined variable 'dfs_oclass'